exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/common/HashPartition.java

```diff
@@ -0,0 +1,579 @@
+/*
```

Comment:
This class should be in the **org.apache.physical.impl.join** package since it is specific to hashjoin.

Reply:
We did raise the option of adopting it for the Hash-Aggregate as well .....